TabGroup/TabGroup.js

```diff
@@ -116,10 +116,13 @@ const TabBase = kind({
 				return (
 					<div aria-label={children} className={css.labeledIcon} role="region">
 						<ToggleButton
+							className={css.button}
```

Comment:
Instead of adding this, why don't we fix the padding `@agate-togglebutton-padding` by setting it 0 for both skins?
![image](https://user-images.githubusercontent.com/5037429/125234167-1ae88400-e31b-11eb-83a5-2c4d639f7290.png)

Reply:
👍